cmd/monaco/integrationtest/v2/config_restore_e2e_test.go

```diff
@@ -383,3 +383,14 @@ func cleanupDeployedConfiguration(t *testing.T, fs afero.Fs, manifestFilepath st
 
 	integrationtest.CleanupIntegrationTest(t, fs, manifestFilepath, loadedManifest, testSuffix)
 }
+
+func TestRestoreConfigs_FromDownloadWithPlatformManifestFile_withAutomation(t *testing.T) {
```

Comment:
I'd place this new method with the others on top of the file - the organization seems to be test cases on top, helper/shared methods below

Reply:
done